records/generator.go

```diff
@@ -111,7 +111,13 @@ type EnumerationData struct {
 
 // NewRecordGenerator returns a RecordGenerator that's been configured with a timeout.
 func NewRecordGenerator(httpTimeout time.Duration) *RecordGenerator {
-	rg := &RecordGenerator{httpClient: http.Client{Timeout: httpTimeout}}
+	enumData := EnumerationData{
+		Frameworks: []*EnumerableFramework{},
```

Comment:
can't you just use a json `omitempty` tag for `Frameworks` in the struct where it's defined?